probe/host/host.go

```diff
@@ -77,15 +78,21 @@ func (s *Server) Config(gConf global.ProbeSettings) error {
 	// 4. retrieve the cpu core:		`grep -c ^processor /proc/cpuinfo;`
 	// 5. retrieve the cpu usage:	`top -b -n 1 | grep Cpu | awk -F ":" '{print $2}'`
 	//    output example: 1.6 us,  0.0 sy,  0.0 ni, 98.4 id,  0.0 wa,  0.0 hi,  0.0 si,  0.0 st
-	// 6. retrieve the disk usage	`df -h / 2>/dev/null | awk '$NF=="/"{printf "%d %d %s\n", $3,$2,$5}'`
-	//    output: used(GB) total(GB) usage(%), example: 40 970 5%
+	// 6. retrieve the disk usage	`df -h / 2>/dev/null | awk '$NF=="/"{printf "%d %d %s\n", $3,$2,$5,$6}'`
+	//    output: used(GB) total(GB) usage(%) disk, example: 40 970 5% /
 
 	s.Command = `hostname;
 	awk -F= '/^NAME/{print $2}' /etc/os-release | tr -d '\"';
 	free -m | awk 'NR==2{printf "%s %s %.2f\n", $3,$2,$3*100/$2 }';
 	grep -c ^processor /proc/cpuinfo;
-	top -b -n 1 | grep Cpu | awk -F ":" '{print $2}';
-	df -h / 2>/dev/null | awk '$NF=="/"{printf "%d %d %s\n", $3,$2,$5}'`
+	top -b -n 1 | grep Cpu | awk -F ":" '{print $2}';` + "\n"
+
+	if len(s.Disks) == 0 {
+		s.Disks = []string{"/"}
+	}
+	for _, disk := range s.Disks {
+		s.Command += "\t" + `df -h ` + disk + ` 2>/dev/null | awk '$NF=="` + disk + `"{printf "%d %d %s %s\n", $3,$2,$5,$6}';` + "\n"
```

Comment:
We can implode the disks into a single command such as `dh -h / /data` and adapt the awk accordingly
```shell
df / /media 2>/dev/null | awk '(NR>1){printf "%d %d %s %s\n", $3,$2,$5,$6}'
301876116 459923528 70% /
666395536 960385056 74% /media
```

Reply:
Good, let me change the code.